sw/device/silicon_creator/lib/manifest_def.c

```diff
@@ -28,7 +28,7 @@ static manifest_t kManifest_ = {
     .manifest_version =
         (manifest_version_t){
             .minor = kManifestVersionMinor1,
-            .major = kManifestVersionMajor1,
+            .major = kManifestVersionMajor2,
```

Comment:
I'm not sure we should advance the version number just yet -- even though we've defined what the manifest structure is with ECDSA keys, we aren't actually using them yet.

Reply:
Agreed. I reverted this change. Will update once we enable ECDSA verify.